api/user.go

```diff
@@ -1510,6 +1510,8 @@ func UpdateActive(c *Context, user *model.User, active bool) *model.User {
 			RevokeAllSession(c, user.Id)
 		}
 
+		<-Srv.Store.Channel().ExtraUpdateByUser(user.Id, model.GetMillis())
+
```

Comment:
I think we should check for the error and set `c.Err; return` so it will get printed.